### ISSUE TITLE
can't set env on a rack

### ISSUE THREAD

**Issue by contra** [convox/rack#2123]
I've got two racks - both in the running state.

Ran the same command, one worked - the other failed and will not work no matter how many times I try.

```
convox env set CONVOX_KEY=test
Updating environment... ERROR: unable to set environment on app: staeco
```

There doesn't seem to be any real way to reboot a rack, that's all I can think of trying at this point. All racks and CLI are latest as of posting.

Semi-related: Can we get error messages on the CLI/API responses? "unable to build" "unable to set" "unable to X" seems to be a recurring theme and leaves people in the dark, opening GH issues, for something that's usually basic.

**Comment by contra** [comment]
Okay - found the issue (only visible on the convox web portal)

App was in a failed state - that info wasn't very visible. Logging into cloudformation and finishing the rollback, then trying again fixed the problem.

**Comment by contra** [comment]
I think this is solved by #2117 and released, closing
